### ISSUE TITLE
Some custom trained models do not work

### ISSUE THREAD

**Issue by Wayne-nulitics** [EnoxSoftware/YOLOv8WithOpenCVForUnityExample#2]
When I train a model and test it in Python it works, if I then use the same .onnx file in Unity it does not detect the objects.

**Comment by Wayne-nulitics** [comment]
I tested the onnx model with Unity Barracude v3 and it also works there, so definatly a problem with OpenCV Yolov8 implementation

**Comment by RoadwayVR** [comment]
@Wayne-nulitics have you fixed the issue?

**Comment by Wayne-nulitics** [comment]
> @Wayne-nulitics have you fixed the issue?

@RoadwayVR yes, I'm using the new new version 1.05, I think, the training is using Ultralytics Yolov8, we do all our models and training offline, no cloud services, due to confidentiality of content/images, for this we use PyTorch and then convert the .pt file to .onnx version 12, this works well in Yolov8 from OpenCV

**Comment by Wayne-nulitics** [comment]
Solved by using new version 1.05 of example and Ultralytics YOLOv8.1.30

**Comment by dmm489** [comment]
What did you change in the scripts to get it working on a custom model? I'm trying to do it myself but am getting errors. I swapped the .onnx and coco.names files and changed the num_classes variable in ObjectDetector.cs but I'm still having trouble. Any help would be appreciated
